### ISSUE TITLE
Community Chart Suggestion: SpeedTest

### ISSUE THREAD

**Issue by pabloeisenhut** [truenas/apps#336]
https://github.com/openspeedtest/Speed-Test

**Comment by RemyDG0** [comment]
I agree this package is very practical, and was available via TrueCharts so it is perfectly compatible with TrueNas Scale

**Comment by mgittelman** [comment]
I would also like to see this added. I was previously able to install this with TrueCharts and also have it on several Core installs from the community list. Very useful.

**Comment by S1lencio** [comment]
Added this as a custom app today, works as expected in both http and https mode. Do note: the image is not `openspeedtest`, but `openspeedtest/latest`. The tag is still latest.

I would still love for this to be added as well, this app is incredibly simple, it only requires two ports.
